Cognite.Simulator.Utils/ConnectorBase.cs

```diff
@@ -212,50 +212,41 @@ protected async Task InitRemoteSimulatorIntegrations(CancellationToken token)
-                    if (existing == null)
-                    {
-                        _logger.LogInformation("Creating new simulator integration for {Simulator}", simulator.Name);
-                        var existingSimulators = await Cdf.CogniteClient.Alpha.Simulators.ListAsync(
-                            new SimulatorQuery (),
-                            token).ConfigureAwait(false);
-                        var existingSimulator = existingSimulators.Items.FirstOrDefault(s => s.ExternalId == simulator.Name);
-
-                        if (existingSimulator == null)
-                        {
-                            _logger.LogWarning("Simulator {Simulator} not found in CDF", simulator.Name);
-                            throw new ConnectorException($"Simulator {simulator.Name} not found in CDF");
-                        }
 
-                        var integrationToCreate = new SimulatorIntegrationCreate
-                        {
-                            ExternalId = connectorName,
-                            SimulatorExternalId = simulator.Name,
-                            DataSetId = simulator.DataSetId,
-                            ConnectorVersion = GetConnectorVersion() ?? "N/A",
-                            SimulatorVersion = GetSimulatorVersion(simulator.Name) ?? "N/A",
-                        };
-
-                        var res = await simulatorsApi.CreateSimulatorIntegrationAsync(new List<SimulatorIntegrationCreate> {
-                            integrationToCreate
-                        }, token).ConfigureAwait(false);
-                        _simulatorIntegrationsList[simulator.Name] = res.First();
-                    }
-                    else
+                    var integrationToCreate = new SimulatorIntegrationCreate
                     {
-                        _logger.LogInformation("Found existing simulator integration for {Simulator}", simulator.Name);
-                        _simulatorIntegrationsList[simulator.Name] = existing;
-                    }
+                        ExternalId = connectorName,
+                        SimulatorExternalId = simulatorExternalId,
+                        DataSetId = _config.DataSetId,
+                        ConnectorVersion = GetConnectorVersion() ?? "N/A",
+                        SimulatorVersion = GetSimulatorVersion(simulatorExternalId) ?? "N/A",
+                    };
+
+                    var res = await simulatorsApi.CreateSimulatorIntegrationAsync(new List<SimulatorIntegrationCreate> {
+                        integrationToCreate
+                    }, token).ConfigureAwait(false);
+                    _simulatorIntegrationsList[simulatorExternalId] = res.First();
+                }
+                else
+                {
+                    _logger.LogInformation("Found existing simulator integration for {Simulator}", simulatorExternalId);
+                    _simulatorIntegrationsList[simulatorExternalId] = existing;
```

Comment:
would there be a need for this list now that we only have one integration?

Reply:
agree